config/types/update.go

```diff
@@ -0,0 +1,100 @@
+// Copyright 2016 CoreOS, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package types
+
+import (
+	"errors"
+	"fmt"
+	"net/url"
+	"strings"
+
+	ignTypes "github.com/coreos/ignition/config/v2_0/types"
+	"github.com/coreos/ignition/config/validate/report"
+	"github.com/vincent-petithory/dataurl"
+)
+
+var (
+	ErrUnknownChannel  = errors.New("unknown update channel")
+	ErrUnknownStrategy = errors.New("unknown reboot strategy")
+)
+
+type Update struct {
+	Channel  UpdateChannel  `yaml:"channel"`
+	Strategy UpdateStrategy `yaml:"strategy"`
```

Comment:
Let's go ahead and remove the strategy. This should be under a locksmith section instead.